tests/unit/pages/subjects/Subjects.spec.jsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import Subjects from '~/pages/subjects/Subjects'
+import { ModalProvider } from '~/context/modal-context'
+
+
+vi.mock('~/context/modal-context', () => ({
+  useModalContext: () => ({
+    closeModal: vi.fn()
+  }),
+  ModalProvider: vi.fn(({ children }) => <div>{children}</div>)
+}))
+
+vi.mock('~/components/page-wrapper/PageWrapper', () => {
+  const PageWrapperMock = ({ children }) => <div>{children}</div>
+  return {
+    __esModule: true,
+    default: vi.fn().mockImplementation(PageWrapperMock)
+  }
```

Comment:
It should be work easier 
```suggestion
  return {
    __esModule: true,
    default:  ({ children }) => <div>{children}</div>
  }
```